pydantic/color.py

```diff
@@ -70,11 +70,13 @@ def __init__(self, value: ColorType) -> None:
             self._rgba = parse_tuple(value)
         elif isinstance(value, str):
             self._rgba = parse_str(value)
+        elif isinstance(value, Color):
+            self._rgba = value._rgba
         else:
-            raise ColorError(reason='value must be a tuple, list or string')
+            raise ColorError(reason='value must be a tuple, list, string or pydantic.color.Color')
```

Comment:
I think no need to change this, I think it's expected and this could be a confusing message to end users.

Reply:
Gotcha - changing now.